utils.js

```diff
@@ -0,0 +1,31 @@
+const bitcoin = require("bitcoinjs-lib");
+
+const btcAddressFromPublicKeyHash = (pubKeyHash, network) => {
+    return bitcoin.payments.p2pkh({
+        hash: Buffer.from(stripHexPrefix(pubKeyHash), "hex"),
+        network: getNetwork(network),
+    }).address;
+};
+
+const getNetwork = (network) => {
+    return network === "mainnet" ? bitcoin.networks.MAINNET : bitcoin.networks.testnet;
```

Comment:
Did you test the tool against mainnet? I believe the name for mainnet network is `bitcoin`. See https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/networks.d.ts

Reply:
Yes I Did Test It With Mainnet

Reply:
For Some Weird Reason, It Works Well With bitcoin.networks.MAINNET or bitcoin.networks.mainnet or bitcoin.networks.bitcoin, The Latter Is The Correct One

Reply:
It probably defaults to mainnet